Cargo.toml

```diff
@@ -62,7 +62,7 @@ path_abs = { version = "0.5", default-features = false }
 clircle = "0.4"
 bugreport = { version = "0.5.0", optional = true }
 dirs = { version = "5.0.0", optional = true }
-grep-cli = { version = "0.1.7", optional = true }
+grep-cli = { version = "0.1.9", optional = true }
```

Comment:
This requires Rust 1.70 and I believe such a change shouldn't be done in this PR...